### ISSUE TITLE
fatal: unsafe repository ('/github/workspace' is owned by someone else)

### ISSUE THREAD

**Issue by ravinaik1312** [beatlabs/delete-old-branches-action#20]
A recent GitHub CVE has caused a lot of GitHub Actions including this one to fail with the following error

```
fatal: unsafe repository ('/github/workspace' is owned by someone else)
To add an exception for this directory, call:

	git config --global --add safe.directory /github/workspace
```

Example workflow

```
stale-branches:
    name: Cleanup stale branches
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Run delete-old-branches-action
        uses: beatlabs/delete-old-branches-action@v0.0.6
        with:
          repo_token: ${{ github.token }}
          date: '1 year ago'
          dry_run: false
          extra_protected_branch_regex: ^(Release|production)$

```

**Comment by dbudim** [comment]
Hi)
When will be a new version with this fix?)

**Comment by pkossyfas** [comment]
Hi @dbudim , tag v0.0.9 has been released with this fix (provided by @ravinaik1312 in pr #21)

**Comment by ravinaik1312** [comment]
> Hi @dbudim , tag v0.0.9 has been released with this fix (provided by @ravinaik1312 in pr #21)

@pkossyfas I think the release action that was triggered after that PR failed which is why the new version is not available. I added another PR to fix that GitHub action #22, I'm not sure if that would trigger a new release or re-triggering that GitHub action should make `0.0.9` available.

**Comment by pkossyfas** [comment]
> > Hi @dbudim , tag v0.0.9 has been released with this fix (provided by @ravinaik1312 in pr #21)
> 
> @pkossyfas I think the release action that was triggered after that PR failed which is why the new version is not available. I added another PR to fix that GitHub action #22, I'm not sure if that would trigger a new release or re-triggering that GitHub action should make `0.0.9` available.

Indeed the action that creates the release from the latest tag is failing (due to the same issue with the unsafe repository - which we will fix it [here](https://github.com/beatlabs/release-changelog-action/pull/3)), so for now I have published manually the new release (v0.0.9)

**Comment by dbudim** [comment]
Awesome!
Thank you guys)
